lib/http-bridge/cardano-wallet-http-bridge.cabal

```diff
@@ -65,6 +65,7 @@ library
       Cardano.Wallet.HttpBridge.Transaction
       Data.Packfile
       Servant.Extra.ContentTypes
+      Paths_cardano_wallet_http_bridge
```

Comment:
Unrelated, but had no idea [this](https://stackoverflow.com/questions/33176905/is-there-a-way-to-get-version-of-cabal-package-in-source-code-of-the-package/33176906) was possible 😮